### PR TITLE
V14.0

### PR DESCRIPTION
**Pull request by raghu-madat**

### CODE REVIEW

**Review by vickum — APPROVED** [review]

**Review by shaiknayabrasool-s — APPROVED** [review]

File: docs/reference/Data/ClientCallback.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+##### Data Type : Security
+##### ClientCallback - ClientCallback  data structure.
+---
+```
+#include <bsafe.h>
+```
+
+**Definition :**
+```
+typedef BOOL (*ClientCallback) (char *pAzp, void *vpCallerContext);
```
Comment (saravananbalakris) [inline docs/reference/Data/ClientCallback.md:10]: Is there detailed description about the clientcallback type.

File: docs/howto/user_guide/New_Features_in_HCL_C_API_Toolkit_for_Notes_Domino_14.0.0.md [hunk 2]
```diff
@@ -5,19 +5,34 @@ This section lists the new and changed features in the HCL C API Toolkit for Not
 <b><font size="5" color="#000080"> </font></b><br>
 <b><font size="5" color="#000080">Security Functions</font></b><br>
 <br>
-Some new functions of security have been added to kfm.h.<br>
+Some new  functions of security have been added to kfm.h.<br>
 
 <ul type="disc">
-<li><b>SECKFMGetIDFlags</b><font size="4"> </font>	<b>- </b>Get ID flags information associated to keyfile context<b>.</b>
```
Comment (saravananbalakris) [inline docs/howto/user_guide/New_Features_in_HCL_C_API_Toolkit_for_Notes_Domino_14.0.0.md:11]: Add "-" post function name

File: docs/reference/Data/ResourceCallback.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+##### Data Type : Security
+##### ResourceCallback - ResourceCallback  data structure.
+---
+```
+#include <bsafe.h>
+```
+
+**Definition :**
+```
+typedef BOOL (*ResourceCallback) (char *pszAudience,void *vpCallerContext);
```
Comment (saravananbalakris) [inline docs/reference/Data/ResourceCallback.md:10]: Possible add detailed description

File: docs/reference/Func/SECValidateAccessToken.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+*pszProviderURL  -  Base URL of trusted OIDC provider for this connection.
+
+*pszRequiredScope  -  "Domino.user.all" or equivalent
+
+*pszResourceURL  -  Expected value in audience (aud) claim, such as https://www.example.com.
+
+dwFlags  -  Modify behavior from defaults. See fJWT_validate_* below.
+
+*vpOptionalParams  -  pszCustomClaimName.
```
Comment (saravananbalakris) [inline docs/reference/Func/SECValidateAccessToken.md]: Not clear on this pszCustomClaimName & ResourceCallback in next line

**Review by saravananbalakris — CHANGES_REQUESTED** [review]

File: docs/reference/Data/ClientCallback.md (hunk 1 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Data/ClientCallback.md:10]: fixed.

File: docs/howto/user_guide/New_Features_in_HCL_C_API_Toolkit_for_Notes_Domino_14.0.0.md (hunk 2 above)
Comment (shaiknayabrasool-s) [reply docs/howto/user_guide/New_Features_in_HCL_C_API_Toolkit_for_Notes_Domino_14.0.0.md:11]: Corrected

File: docs/reference/Data/ResourceCallback.md (hunk 3 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Data/ResourceCallback.md:10]: added

File: docs/reference/Func/SECValidateAccessToken.md (hunk 4 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Func/SECValidateAccessToken.md]: Corrected

**Pull request merged**